randovania/data/json_data/prime1.txt

```diff
@@ -1896,44 +1908,104 @@ Asset id: 2993688902
 > Dock to Gully; Heals? False
   > Gully/Dock to Landing Site
       Trivial
+  > Landing Site/Dock to Canyon Cavern
+      Trivial
+  > Landing Site/Dock to Temple Hall
+      Trivial
+  > Landing Site/Dock to Alcove
+      Trivial
+  > Landing Site/Dock to Waterfall Cavern
+      Trivial
 
 > Dock to Canyon Cavern; Heals? False
   > Canyon Cavern/Dock to Landing Site
       Trivial
+  > Landing Site/Dock to Gully
+      Trivial
+  > Landing Site/Dock to Temple Hall
+      Trivial
+  > Landing Site/Dock to Alcove
+      Space Jump Boots
+  > Landing Site/Dock to Waterfall Cavern
+      Trivial
 
 > Dock to Temple Hall; Heals? False
   > Temple Hall/Dock to Landing Site
       Trivial
+  > Landing Site/Dock to Gully
+      Trivial
+  > Landing Site/Dock to Canyon Cavern
+      Trivial
+  > Landing Site/Dock to Alcove
+      Space Jump Boots
+  > Landing Site/Dock to Waterfall Cavern
+      Trivial
 
 > Dock to Alcove; Heals? False
   > Alcove/Dock to Landing Site
       Trivial
+  > Landing Site/Dock to Gully
+      Trivial
+  > Landing Site/Dock to Canyon Cavern
+      Trivial
+  > Landing Site/Dock to Temple Hall
+      Trivial
+  > Landing Site/Dock to Waterfall Cavern
+      Trivial
 
 > Dock to Waterfall Cavern; Heals? False
   > Waterfall Cavern/Dock to Landing Site
       Trivial
+  > Landing Site/Dock to Gully
+      Trivial
+  > Landing Site/Dock to Canyon Cavern
+      Trivial
+  > Landing Site/Dock to Temple Hall
+      Trivial
+  > Landing Site/Dock to Alcove
+      Space Jump Boots
+
+> Pickup (Missile Expansion); Heals? False
```

Comment:
No other node connects to this one, so it's unobtainable.

It's also good to have a node for Save Stations. Right now call them "Save Station" since that's the code for finding save stations.

Also, instead of connecting everything to everything, you could make less connections. So for example, instead of having a path from this missile to all doors, you could just have one to the save station/ship.

Reply:
for now I've done only from alcove to life grove. i'll see tomorrow to solve this and work some more on it